emojify.py

```diff
@@ -5,16 +5,20 @@
 EMOJI_BLOCK_SZ = 32
 
 def main(argc, argv):
-    if argc < 4:
-        print '%s <input gif> <output directory> <emoji name> <max width emojis>' % argv[0]
+    if argc < 5:
```

Comment:
This should still be 4 or the argument will be considered a requirement

Reply:
Your code allows to use 3 arguments instead of 4 (it causes an exception) (at least on linux)